DigitalLearningSolutions.Data.Migrations/DigitalLearningSolutions.Data.Migrations.csproj

```diff
@@ -35,7 +35,7 @@
 
   <ItemGroup>
     <EmbeddedResource Update="Properties\Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
Why is this change needed?

Reply:
This is part of the file added when I make the update I did not manually alter this